spec/helpers/dor_object_helper_spec.rb

```diff
@@ -48,4 +48,31 @@
       expect(render_status_style(nil, nil)).to eq('')
     end
   end
+
+  describe 'get_metadata_source' do
+    it 'should return Metadata Toolkit when identityMetadata.otherId contains mdtoolkit' do
+      id_metadata = double('identity_metadata')
+      mock_obj = double('Dor::Item', { :identityMetadata => id_metadata })
```

Comment:
First 2 lines are shared in each `it` block.  Probably could go into a `before` block, setting `@id_metadata` and `@mock_obj`.